pan3d/ui/main_drawer.py

```diff
@@ -0,0 +1,71 @@
+from trame.widgets import html, vuetify
+
+
+class MainDrawer(html.Div):
+    def __init__(self):
```

Comment:
Looks like fully static code. No parametrization at all.

Reply:
Yes, while it isn't the typical use case for a component class, I would still want to avoid dissolving this into the main class. I am kind of treating it as function-based code simply to avoid a very large file and to have some conceptual separation of this code. 
If passing in state var names helps to legitimize this separation, I can do that. The expectation is that these var names will never have any other value, so I could put the expected string as defaults, so the API where I've used these "classes" wouldn't have to change either. But it would still technically be possible to reuse the class with different state var names. Is this an acceptable approach?

Reply:
@jourdain Does the approach in [5b7331b](https://github.com/Kitware/pan3d/pull/28/commits/5b7331b4dadd55dcaaa3d2650ee192bb603740af) make this better?

Reply:
yes, perfect!